lib/stores/FileStore.ts

```diff
@@ -32,16 +34,19 @@ const log = debug('tus-node-server:stores:filestore')
 export default class FileStore extends DataStore {
   directory: string
   configstore: Store
+  expirationPeriodInMinutes: number
 
-  constructor({directory, configstore}: Options) {
+  constructor({directory, configstore, expirationPeriodMinutes}: Options) {
     super()
     this.directory = directory
     this.configstore = configstore ?? new Configstore(`${pkg.name}-${pkg.version}`)
+    this.expirationPeriodInMinutes = expirationPeriodMinutes ?? -1
```

Comment:
not sure if we want this in minutes instead of miliseconds?

Reply:
I changed the option to represent milliseconds instead of minutes. 

Also, the reason I am computing `const expires = new Date(creation.getTime() + this.getExpiration())` with the `creation` time instead of storing expiration date is because the expiration period can change over time, "If the expiration is known at the creation, the Upload-Expires header MUST be included in the response to the initial POST request. Its value MAY change over time."

I figured it would be easier to compute the expiration time on the fly with the most current value for `expirationPeriodInMilliseconds` than to save an outdated date in the configstore.